accessibility_scanning/README.md

```diff
@@ -0,0 +1,131 @@
+
+### Installation and setup(For MacOS)
+
+1. [Concourse CI](https://concourse.ci/index.html) allows you to run multiple compliance scanning jobs  on your machine using a virtual machine. It is highly recommended to go through its [excellent documentation and "Hello World" tutorial ](https://concourse.ci/hello-world.html) before writing custom jobs for your project.
+
+2. If you want to run locally Pa11y-ci you will need to :
+
+   * install **vagrant** and **virtualbox** ](http://sourabhbajaj.com/mac-setup/Vagrant/README.html)
+   * use [The Fly CLI](https://concourse.ci/fly-cli.html) , a command line tool for Concourse which is  used for a number of tasks from connecting to a shell in one of your build's containers to uploading new pipeline configuration into a running Concourse.
+
+3. If you're on MacOS , you will have to `chmod +x` the downloaded binary and put it in your `$PATH`. This can be done in one fell swoop with `install path/to/fly /usr/local/bin/fly`
+
+4. Run Vagrant
+
+   ```sh
+   vagrant init concourse/lite # creates ./Vagrantfile
+   vagrant up                  # downloads the box and spins up the VM
+   ```
+
+5. Login into fly
+
+   `fly -t lite login -c http://192.168.100.4:8080`
+
+   ​
+
+### Running Pa11y-ci on your local machine
+
+1. Create a `.yml` file containing the job with tasks that you want to run
+
+`touch accessibility-scan.yml`
+
+`vim accessibility-scan.yml`
+
+```yml
+jobs:
+  - name: accessibility-scan
+    plan:
+    - task: run-pa11y
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: node
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            npm install -g pa11y-ci
+            npm install -g phantomjs
+            mkdir accessibility-scan
+            pa11y-ci --json --sitemap https://18f.gsa.gov/sitemap.xml > accessibility-scan/summary.txt
+            cat accessibility-scan/summary.txt
+```
+
+`type` is almost always going to be `docker-image` . `pa11y-ci` requires node, so we are using a `node` docker-image.
+
+`run` section contains a series of shell command that we are executing:
+
+* install pa11y-ci  and phantomjsfrom the node package manager(npm)
+
+* making a directory and telling pa11y-ci to pipe the results of the scan into a `json` file
+
+  the command is like so `pa11y-ci --json --sitemap https://[18f-static-site-url]/sitemap.xml > dir/file`
+
+* printing the results in our CLI `cat…` (for debugging purposes, optional)
+
+**Note that** we are using the `sitemap.xml`  file instead of individual files(much more efficient)
+
+2. Run
+
+```shell
+fly -t lite set-pipeline -p accessibility-scan -c accessibility-scan.yml
+```
+
+3. Go to the `URL` displayed on your CLI. **Click the plus sign on top right corner and toggle side bar and press play on your pipeline**. Pipelines are posed by default.
```

Comment:
@cmc333333 I am not too experienced enough with docker , but I believe it would. Concourse CI provides [4 different ways to install it locally](https://concourse.ci/installing.html) including with Docker. We use the Vagrant option since it seemed like the least complicated one(and was recommended).